torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3333,6 +3333,12 @@ def _maybe_evaluate_static(
                     subst[sympy.Le(-e.lhs, -e.rhs)] = sympy.true
                     subst[sympy.Lt(e.lhs, e.rhs)] = sympy.false
                     subst[sympy.Lt(-e.lhs, -e.rhs)] = sympy.false
+                elif isinstance(e, sympy.Lt):
+                    subst[sympy.Lt(e.lhs, e.rhs)] = sympy.true
+                    subst[sympy.Le(e.lhs, e.rhs)] = sympy.true
+                    subst[sympy.Gt(e.lhs, e.rhs)] = sympy.false
+                    subst[sympy.Ge(e.lhs, e.rhs)] = sympy.false
+                    subst[sympy.Ne(e.lhs, e.rhs)] = sympy.true
```

Comment:
We should probably canonicalise `Ne` into `Or(Lt(a,b), Lt(b, a))`, but I guess this is fine for now.